bedrock/firefox/templates/firefox/new/base.html

```diff
@@ -10,18 +10,33 @@
 
 {% block page_title_prefix %}{{_('Download Firefox')}} — {% endblock %}
 
-{# Bug 1438302 Avoid duplicate content for en-CA and en-GB pages. #}
-{%- block page_title -%}
-  {% if LANG == 'en-CA' %}
+
+{% if switch('firefox_new_title_experiment') %}
+    {# Bug 1438302 Avoid duplicate content for en-CA and en-GB pages. #}
+    {%- block page_title -%}
+    {% if LANG == 'en-CA' %}
+    The Fast and Free Web Browser 🦊 (CA)
+    {% elif LANG == 'en-GB' %}
+    The Fast and Free Web Browser 🦊 (UK)
+    {% else %}
+    {{_('The Fast and Free Web Browser 🦊')}}
+    {% endif %}
+    {%- endblock -%}
+    {% block page_title_suffix %}{% endblock %}
+    {% block page_desc %}{{_('Download Firefox for Windows, Mac, Linux, Android, and iOS. Firefox is an independent, people-first browser made by Mozilla. Get it now, it's free and fast. ✓')}}{% endblock %}
+{% else %}
+    {# Bug 1438302 Avoid duplicate content for en-CA and en-GB pages. #}
+    {%- block page_title -%}
```

Comment:
You can't include a block twice in a template like this. You will need to do the switch conditonal inside each block. I'll work on an example.